### PR TITLE
Updating health.py

### PR DESCRIPTION
**Pull request by rhiGriff**
**What is the context of this PR?**

Link to Trello card
https://trello.com/c/TR3wcQ0r 

Describe what you have changed and why, link to other PRs or Issues as appropriate.

**How to review**

Any notes for the reviewer  (include screenshots if appropriate).

### CODE REVIEW

File: app/resources/health.py [hunk 1]
```diff
@@ -40,9 +41,17 @@ def get():
...
-        details = {'SMS Log level': settings.SMS_LOG_LEVEL,
+        details = {'Name': settings.NAME,
+                   'Version': settings.VERSION,
+                   'SMS Log level': settings.SMS_LOG_LEVEL,
                    'APP Log Level': settings.APP_LOG_LEVEL,
                    'Database URL': current_app.config['SQLALCHEMY_DATABASE_URI'],
-                   'API Functionality': func_list}
+                   'API Functionality': func_list,
+                   'Using party service mock': party._use_mock,
+                   'SM JWT SECRET': settings.SM_JWT_SECRET,
```
Comment (andrewmil) [inline app/resources/health.py]: Do we want to have an endpoint which returns secrets like this?

**Comment by andrewmil** [comment]
Also one test failing

**Comment by rhiGriff** [comment]
Been fixing test, waiting on the other PR before this one can be completed.

**Comment by rhiGriff** [comment]
Waiting for TS 235 before completing

File: app/resources/health.py (hunk 1 above)
Comment (rjweeks) [inline app/resources/health.py]: No, delete this one now.

**Pull request merged**